docs/m4t/README.md

```diff
@@ -39,8 +39,199 @@ We provide the extensive evaluation results of seamlessM4T-Large and SeamlessM4T
 The evaluation data ids for FLEURS, CoVoST2 and CVSS-C can be found [here](https://dl.fbaipublicfiles.com/seamless/metrics/evaluation_data_ids.zip)
 
 
-## Evaluating SeamlessM4T models
-To reproduce our results, or to evaluate using the same metrics over your own test sets, please check out the [Evaluation README here](../../src/seamless_communication/cli/m4t/evaluate/README.md).
+## Using SeamlessM4T models
+
+### `m4t_predict` with CLI:
+Inference is run with the CLI, from the root directory of the repository.
+
+The model can be specified with `--model_name` `seamlessM4T_v2_large`, `seamlessM4T_large` or `seamlessM4T_medium`:
+
+```bash
+# S2ST:
+m4t_predict <path_to_input_audio> --task s2st --tgt_lang <tgt_lang> --output_path <path_to_save_audio> --model_name seamlessM4T_large
+# S2T:
+m4t_predict <path_to_input_audio> --task s2tt --tgt_lang <tgt_lang>
+# T2TT:
+m4t_predict <input_text> --task t2tt --tgt_lang <tgt_lang> --src_lang <src_lang>
+# T2ST:
+m4t_predict <input_text> --task t2st --tgt_lang <tgt_lang> --src_lang <src_lang> --output_path <path_to_save_audio>
+# ASR:
+m4t_predict <path_to_input_audio> --task asr --tgt_lang <tgt_lang>
+```
+
+The input audio must be 16kHz currently. Here's how you could resample your audio:
+```python
+import torchaudio
+resample_rate = 16000
+waveform, sample_rate = torchaudio.load(<path_to_input_audio>)
+resampler = torchaudio.transforms.Resample(sample_rate, resample_rate, dtype=waveform.dtype)
+resampled_waveform = resampler(waveform)
```

Comment:
Could we do this one-liner instead? https://github.com/facebookresearch/seamless_communication/blob/main/src/seamless_communication/cli/expressivity/predict/predict.py#L115